### ISSUE TITLE
Chatito npm script generates training/testing data only from last .chatito file

### ISSUE THREAD

**Issue by heaven00** [rodrigopivi/Chatito#31]
I have a list of `.chatito` files in the folder and I am using the following command

```
npx folder/path --format rasa --outputPath data/nlu
```

it prints 

```
Processing file: /folder/path/bye.chatito
Processing file: /folder/path/greet.chatito
Processing file: /folder/path/negative.chatito
Saved training dataset: ./rasa_dataset_training.json
Saved testing dataset: ./rasa_dataset_training.json
```

But the training set and test set only contains sample from `negative.chatito`, the last processed file.
